ups/eupspkg.cfg.sh

```diff
@@ -13,4 +13,4 @@ export LUA_LIBS="-L$LUA_DIR/lib -llua"
 # Note: It is assumed that glib2 is installed system-wide and that
 # ./configure will detect it up via pkg-config.
 
-CONFIGURE_OPTIONS="--prefix=$PREFIX --with-lua --with-mysql=$MYSQL_DIR/bin/mysql_config"
+CONFIGURE_OPTIONS="--prefix=$PREFIX --with-lua --with-mysql=$MARIADB_DIR/bin/mysql_config"
```

Comment:
on the qserv side you kept "MYSQL_DIR" name. I think it'd be better to keep things consistent, otherwise we'll be always running into the issue "should I use "mariadb_*" or "mysql_*"" name here? So, I propose we keep MYSQL_DIR name.